c2corg_ui/templates/profile/helpers/view.html

```diff
@@ -56,7 +56,7 @@
       <p class="float-button-text" translate>Add images</p>
     </div>
 
-    <div class="dropdown" ng-if="userCtrl.hasEditRights('profiles', ${options | n})">
+    <div class="dropdown" ng-if="userCtrl.hasEditRights('profiles', ${options | n}) || userCtrl.isModerator()">
```

Comment:
This is redundant because moderators already have editing rights on profiles:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/auth/authservice.js#L111-L117